arduino-ide-extension/src/browser/serial/monitor/serial-monitor-send-input.tsx

```diff
@@ -6,6 +6,74 @@ import { BoardsServiceProvider } from '../../boards/boards-service-provider';
 import { MonitorModel } from '../../monitor-model';
 import { Unknown } from '../../../common/nls';
 
+class RingList {
```

Comment:
`RingList` implies that message history is a cycle.

With the following sent messages:
 - `one`,
 - `two`, and
 - `three`.

I would expect to get `three` after `one`. But no, so the class name is wrong.

Reply:
True. I debated with myself over that name. I'm changing it to HistoryStack. It's more descriptive.